apps/er-coap/er-coap-observe.c

```diff
@@ -250,6 +248,7 @@ coap_observe_handler(resource_t *resource, void *request, void *response)
            * Uncomment if you want an information about the avaiable observers.
```

Comment:
If you are touching this anyway. Why not put it in an #if 0 .... #endif. - I know its a bit off topic and only a suggestion.